### PR TITLE
feat: initial version of contributing doc

### PR DESCRIPTION
**Pull request by othertea**
This PR contains the initial version of the contributing doc and points the README to it.

### CODE REVIEW

**Comment by pascalnotin** [comment]
Hi @othertea -- this looks good to me! Could you please add an Apache 2.0 license? Ok to merge into main afterwards

**Comment by jamaliki** [comment]
Hey @othertea , this is great! Maybe add a point after the PR about requesting reviews? Or do we want to take charge of that ourselves?

**Comment by othertea** [comment]
@jamaliki thanks for the feedback! I added requesting a review as another step -- let me know if it seems reasonable! I'm happy to update it further.

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Copyright 2023 OpenBioML
```
Comment (NZ99) [inline LICENSE.md:1]: Small nitpick, but I'd advocate in favor of making it more precise in noting that the copyright belongs to repo contributors, e.g. something like "Copyright 2023 the authors". OpenBioML works as well for me though.

File: docs/CONTRIBUTING.md
Comment (NZ99) [inline docs/CONTRIBUTING.md:1]: Probably worth noting after the open issues that contributors should claim issues through the /take command

**Review by NZ99 — COMMENTED** [review]
The rest looks great to me, thank you Karolyn!

File: LICENSE.md (hunk 1 above)
Comment (othertea) [reply LICENSE.md]: Thanks, I've updated it to say "the authors"!

File: docs/CONTRIBUTING.md
Comment (othertea) [reply docs/CONTRIBUTING.md:1]: Thanks, I've added this to the doc!

**Comment by othertea** [comment]
@NZ99 I've made some updates to address your comments, let me know what you think!

**Review by pascalnotin — APPROVED** [review]
On board with all recent changes  - thanks @othertea !

**Pull request merged**